adds the iMCP server to the list of 3rd party MCP servers (#1458)
modelcontextprotocol/servers: README.md

- **[Hyperliquid](https://github.com/mektigboy/server-hyperliquid)** - An MCP server implementation that integrates the Hyperliquid SDK for exchange data.
- **[iFlytek Workflow](https://github.com/iflytek/ifly-workflow-mcp-server)** - Connect to iFlytek Workflow via the MCP server and run your own Agent.
- **[Image Generation](https://github.com/GongRzhe/Image-Generation-MCP-Server)** - This MCP server provides image generation capabilities using the Replicate Flux model.

> - **[iMCP](https://github.com/loopwork-ai/iMCP)** - Connect iMessage, Reminders, Mail and other Apple services via this native MacOS app and MCP server. 

Reviewer: ```suggestion
- **[iMCP](https://github.com/loopwork-ai/iMCP)** - A macOS app that provides an MCP server for your iMessage, Reminders, and other Apple services. 
```
